app/api/search/search.js

```diff
@@ -416,10 +416,12 @@ const getInheritedEntitiesData = async (toFetchByTemplate, language, user) =>
       }
       return entities.get(query, {
         ...toFetchByTemplate[t].properties.reduce(
-          (memo, n) => Object.assign(memo, { [`metadata.${n}.value`]: 1 }),
+          (memo, n) =>
+            Object.assign(memo, { [`metadata.${n}.value`]: 1, [`metadata.${n}.label`]: 1 }),
           {}
         ),
         sharedId: 1,
+        title: 1,
```

Comment:
is it possible to use "true" instead of "1" ?

Reply:
According to the documentation, yes, projections can be 1 or true.
https://docs.mongodb.com/v4.0/reference/method/db.collection.find/#projection

I do believe its more semantic.